static/_redirects

```diff
@@ -33,6 +33,9 @@
 /calico-enterprise/3.15/ /calico-enterprise/3.15/about-calico-enterprise 301
 /calico-enterprise/3.14/ /calico-enterprise/3.15/about-calico-enterprise 301
 
+# Redirects for CC pages in Candu
+/calico-cloud/image-assurance/scan-image-registries#start-the-cli-scanner /calico-cloud/latest/image-assurance/overview
```

Comment:
@bmckercher123 the direct target is not correct. 

We want to redirect the user to: 
- https://docs.tigera.io/calico-cloud/image-assurance/scanners/cli-based-scanner#start-the-cli-scanner

It's also not specific to Candu. Candu is only one of the instances where the old URL was used. 

I would label this one as an Image Assurance redirect.

Reply:
Fixed.